airflow/www/templates/appbuilder/navbar_menu.html

```diff
@@ -18,7 +18,12 @@
 #}
 
 {% macro menu_item(item) %}
-  <a href="{{item.get_url()}}">{{_(item.label)}}</a>
+  {% set url = item.get_url() %}
+  {% if 'apache' in url %}
```

Comment:
I think ust `apache` here is not enough I believe we should check for valid URLs  that we know can be used here:

* development docs from s3
* https://github.com/apache/airflow
* https://airflow.apache.org/

Just having `apache` - might accidentally be in someones's URL (not very likely but possible)

Reply:
Thanks @potiuk 
Ok, got this , would you suggest we go explicit  for these two/ three?

something like "if url in [https://github.com/apache/airflow, https://airflow.apache.org/, ]"....

Reply:
My thoughts here..

```

{% if 'https://github.com/apache/airflow' in url or
           'https://airflow.apache.org/' in url or
           'apache-airflow-docs.s3' in url %}

```

or declaring a proper list to make this neater.....

Reply:
Yeah. declaring a list would be better

Reply:
@potiuk , done with this now.

Reply:
@potiuk .. seems it's all set..Can we mark this as resolved..